camel/configs.py

```diff
@@ -63,6 +63,10 @@ class ChatGPTConfig:
         user (str, optional): A unique identifier representing your end-user,
             which can help OpenAI to monitor and detect abuse.
             (default: :obj:`""`)
+        stream (bool, optional): If set, partial message deltas will be sent,
+            like in ChatGPT. Tokens will be sent as data-only server-sent
+            events as they become available, with the stream terminated by
+            a data: [DONE] message. (default: :obj:`False`)
```

Comment:
It seems `stream` already exists in the `ChatGPTConfig`. No need to add it again.